APISamples/powershell/README.md

```diff
@@ -0,0 +1,43 @@
+# Getting started
+This project demonstrates examples to upload/download/delete an item in the Report Server API (August 2017 Preview). 
+
+## Setting up CORS & CSRF
```

Comment:
Remove the CORS part since it only applies for Web Apps, not desktop applications or scripts